kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/operator/KroxyliciousOperatorYamlInstaller.java

```diff
@@ -91,7 +90,7 @@ public KroxyliciousOperatorYamlInstaller(String namespaceInstallTo) {
     @NonNull
     private static List<Path> installFilesMatching(Predicate<Path> matcher) {
         List<Path> crdFiles;
-        try (var fileStream = Files.list(Path.of(Constants.OPERATOR_INSTALL_DIR))) {
+        try (var fileStream = Files.list(Path.of(Environment.KROXYLICIOUS_OPERATOR_INSTALL_DIR))) {
```

Comment:
Setting the right install dir with the deployment file pointing to the proper repositories, avoid the need to replace all images in the file again.

Reply:
Ok, that sounds like a sensible thing todo. 

Where /how do the system tests get an install dir? what is in the install dir? 

I'm partly questioning the naming as the operator is installed on the cluster and not in some dir. Maybe the `KROXYLICIOUS_OPERATOR_INSTALLER_DIR` .

Reply:
Do we need a different abstraction /api here so that we can use OLM to install the operator, when we get round to supporting it?

Reply:
We already have support for OLM installation on `KroxyliciousOperatorOlmBundleInstaller` class. This class in for yaml installation and the installation dir is where all crds and deployment file are located